src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,377 @@
+be used to complete a payment.
+
+Recipients who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery Server to 
+obtain a PayID Easy Checkout URL. PayID Discovery Servers that wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this document
+in all PayID Easy Checkout Discovery responses.
+
+Recipients SHOULD implement fallback measures to complete a checkout flow if a payer's wallet does not support PayID Easy Checkout.
+
+The following steps describe how a PayID Easy Checkout Client can query a PayID Discovery Server to obtain a PayID Easy Checkout URL.
+
+### Step 1: Assemble PayID Easy Checkout Discovery URL
+The process of assembling a PayID Discovery URL is defined in section 4.1.1 of [PAYID-DISCOVERY][], and is
+the same as for this protocol.
+
+### Step 2: Query PayID Easy Checkout Discovery URL
+Querying the PayID Discovery URL is defined in section 4.1.2 of [PAYID-DISCOVERY][], and is performed
+in the same way as this protocol.
+
+Clients SHOULD implement fallback measures to complete checkout if the PayID Easy Checkout Discovery query fails.
+
+### Step 3: Parse PayID Easy Checkout Metadata
+If PayID Easy Checkout is supported, a PayID Discovery server MUST respond with an HTTP status code `200` and a JSON payload
+containing a JSON Resource Descriptor (JRD) that contains a link conforming to the [JRD section](#payid-easy-checkout-jrds) of this document.
+
+For example, a PayID Discovery Server might respond to a PayID Discovery query with the following payload:
+
+     {
+        "subject": "payid:alice$wallet.com",
+        "links": [
+            {
+                "rel" : "https://payid.org/ns/payid-easy-checkout-uri/1.0",
+                "href": "https://wallet.com/checkout"
+            }
+        ]
+     }
+     
+A PayID Easy Checkout client MUST parse this response to find the PayID Easy Checkout Link. 
+If the JRD returned from the PayID Discovery query does not contain a 
+PayID Easy Checkout Link in its 'links' collection, PayID Easy Checkout is considered to have failed.
+Once a PayID Easy Checkout URL has been obtained from the PayID Easy Checkout Link, 
+PayID Easy Checkout Discovery is considered to be complete. 
+
+## PayID Easy Checkout URL Assembly
+A PayID Easy Checkout URL represents the resource on a wallet that can
+be used by a payer to complete a payment. However, before directing a payer to their wallet, the recipient
+MUST append all of the query parameters defined in the [following section](#payid-easy-checkout-url-query-parameters).
+
+Once a PayID Easy Checkout URL is assembled, PayID Easy Checkout is considered to be complete.
+
+### PayID Easy Checkout URL Query Parameters
+This specification defines several query parameter names and corresponding datatypes which MUST be added to the
```

Comment:
Are all the following fields MUST?

Reply:
Yes.  We decided that these fields (and any others that may come in subsequent versions of the spec) are the lowest common denominator that any given wallet would need to populate a transaction, so receivers (merchants) have to include all of them for every wallet